soco/core.py

```diff
@@ -479,6 +480,30 @@ def ramp_to_volume(self, volume, ramp_type='SLEEP_TIMER_RAMP_TYPE'):
         ])
         return int(response['RampTime'])
 
+    def set_relative_volume(self, relative_volume):
+        """Adjust the volume up or down by a relative amount.
+
+        If the adjustment causes the volume to overshoot the maximum value
+        of 100, the volume will be set to 100. If the adjustment causes the
+        volume to undershoot the minimum value of 0, the volume will be set
+        to 0.
+
```

Comment:
The SoCo instance has a `volume` property, which means that for relative changes you can do something like `device.volume += 10`, which has the same result as this BUT which requires 2 network calls. I think the documentation here should mention that option and the difference. You should probably check the other doc strings to see by what name a SoCo instance is usually referred to.

Reply:
Thanks for the comments. I've added documentation to refer to the increment/decrement assignment operators.